src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -17,7 +17,10 @@
 @Singleton
 public class ApplicationParams {
     @Value("#{'${ccd.callback.retries}'.split(',')}")
-    private List<Integer> callbackRetries;
+    private List<Integer> callbackRetryIntervalsInSeconds;
+
+    @Value("${http.client.read.timeout}")
```

Comment:
Should we add a default value to the application.properties?

Reply:
The default value in application.properties is `60000`.

Reply:
Fine. Couldn't find it in the PR as the code was folded and the property was already there.